Week1/homework/queries.js

```diff
@@ -0,0 +1,114 @@
+var mysql      = require('mysql');
```

Comment:
You did not provide any INSERT in your homework, you need some (and I need some) in order to test this queries :)